files/rollup.config.js

```diff
@@ -0,0 +1,36 @@
+import commonjs from 'rollup-plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import pkg from '../package.json';
+
+export default {
+    entry: './rambda.js',
+    plugins: [
+        commonjs(),
+        babel({
+            presets: [['es2015', {modules: false}]],
+            plugins: ['external-helpers'],
+        }),
+    ],
+    targets: [
+        {
+            dest: pkg.main,
+            format: 'cjs',
+            exports: 'named',
+            interop: false,
+            sourceMap: false,

```

Comment:
I disabled source maps because they are not generated currently, but I believe they should be enabled because they provide better development experience.